pkg/webhook/admission/validator.go

```diff
@@ -89,5 +90,19 @@ func (h *validatingHandler) Handle(ctx context.Context, req Request) Response {
 		}
 	}
 
+	if req.Operation == v1beta1.Delete {
+		// In reference to PR: https://github.com/kubernetes/kubernetes/pull/76346
+		// OldObject contains the object being deleted
+		err := h.decoder.DecodeRaw(req.OldObject, obj)
```

Comment:
`req.OldObject` is an empty object when kube-apiserver is v1.14 or earlier.
`obj` will be a object with zero values everywhere.
`DecodeRaw` really should return an error when there is an empty object.
I created #529. We should merge that first.